src/main/java/org/miracum/etl/fhirtoomop/mapper/PatientMapper.java

```diff
@@ -83,8 +85,12 @@ public class PatientMapper implements FhirMapper<Patient> {
    * @param dbMappings collections for the intermediate storage of data from OMOP CDM in RAM
    */
   @Autowired
-  public PatientMapper(IFhirPath fhirPath, Boolean bulkload, DbMappings dbMappings) {
-
+  public PatientMapper(
+      IFhirPath fhirPath,
+      @Qualifier("bulkload") Boolean bulkload,
```

Comment:
What does the `@Qualifier` do?

Reply:
You need to use it when there are multiple beans of the same type

Reply:
Why did it not need it before though?

Reply:
Because they didn't have another Boolean bean when they should have

Reply:
Ohh! Is it smart enough to just do `@Qualifier Boolean bulkload` or `@Qualifier() Boolean bulkload`? Just curious if we can avoid duplicating the name